addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.js

```diff
@@ -77,6 +139,44 @@ export class AvatarCardPopover extends Component {
         this.props.close();
     }
 
+    onClickRemove() {
+        this.dialog.add(ConfirmationDialog, {
+            body: _t(
+                'Do you want to remove "%s" from this channel?',
+                this.props.channelMember?.persona.name
+            ),
+            cancel: () => {},
+            confirm: () =>
+                rpc("/discuss/channel/remove_member", {
+                    member_id: this.props.channelMember.id,
+                }),
+        });
+        this.props.close();
```

Comment:
I don't think we necessarily need to close the avatar card after doing an action.
Especially if we do cancel. If we confirm I guess it's ok as the member will be gone.